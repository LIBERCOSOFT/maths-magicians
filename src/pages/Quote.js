/* eslint-disable max-len */
import React from 'react';

const Quote = () => (
  <div className="quoteContainer">
    <h2>Mathematics is not about numbers, quations, computations, or algorithms; It is about understanding. </h2>
    <h4>- William Paul Thurston</h4>
  </div>
);

export default Quote;
