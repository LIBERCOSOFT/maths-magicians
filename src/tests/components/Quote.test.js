import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { BrowserRouter } from 'react-router-dom';
import Quote from '../../pages/Quote';

test('renders quote component snapshot', () => {
  const { asFragment } = render(<Quote />, { wrapper: BrowserRouter });
  expect(asFragment()).toMatchSnapshot();
});
