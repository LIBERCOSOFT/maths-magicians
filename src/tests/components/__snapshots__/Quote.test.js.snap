// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders quote component snapshot 1`] = `
<DocumentFragment>
  <div
    class="quoteContainer"
  >
    <h2>
      Mathematics is not about numbers, quations, computations, or algorithms; It is about understanding. 
    </h2>
    <h4>
      - William Paul Thurston
    </h4>
  </div>
</DocumentFragment>
`;
